WordPress/gutenberg
# Library - make pattern title clickable

## What?


Makes the title of a pattern in the Library clickable as well as the preview

## Why?


The previews for Patterns in the library can take time to load/render. This means you have to wait to be able to click through to the pattern. It's inherently unstable UX.

By making the pattern _title_ also clickable, we allow users to click through even if:

- the pattern is still loading
- the pattern preview fails to render

I also prefer the UX as I expected the pattern title to be clickable and it wasn't.

## How?


Uses a Button in `link` variant form, to click through to the same route as the preview click through.

## Testing Instructions





- Open Library
- Click on Pattern Preview box. Make note of URL and state of UI.
- Go back to Library.
- Click on title of the _same_ Pattern.
- Check you went to the same place as before.

### Testing Instructions for Keyboard


## Screenshots or screencast 
<img width="1275" alt="Screen Shot 2023-06-26 at 11 13 00 (1)" src="https://github.com/WordPress/gutenberg/assets/444434/4339f4ae-a92c-4c5e-9891-ffd5079d9d6f">
